bundle/manifests/oadp.openshift.io_dataprotectionapplications.yaml

```diff
@@ -1069,5 +1392,5 @@ status:
   acceptedNames:
     kind: ""
     plural: ""
-  conditions: []
-  storedVersions: []
+  conditions: null
```

Comment:
I am not sure how this change would affect us: `[]` -> `null` , maybe its harmless

Reply:
I think its controller gen default that changed

but we also did this change on the other PR https://github.com/openshift/oadp-operator/pull/1398/files#r1617718698

Reply:
@shubham-pampattiwar @mateusoliveira43 Yes, controller-gen upgrade changed the way some of this is rendered. This matches oadp-1.4.